pages/[lng]/review/[id].tsx

```diff
@@ -137,8 +138,15 @@ const ReviewPage: FC<any> = ({
   );
 };
 
-export const getServerSideProps: GetServerSideProps = async ({ params, req }) => {
-  const brand = await useBrand(req)
+export const getServerSideProps: GetServerSideProps = async ({
+  req,
+  res,
+  params
+}) => {
+  const [brand] = await Promise.all([
+    useBrand(req),
+    useAuthToken({ req, res, env: process.env }),
```

Comment:
```suggestion
    useAuthToken({ req, res, env: process.env })
```

koma di akhirnya mungkin bisa dibumihanguskan ini pacc